go/providers.rst

```diff
@@ -303,3 +303,16 @@ each package.
 | * ``data``: list of data ``File``s.                                                              |
 +--------------------------------+-----------------------------------------------------------------+
 
+GoChecker
```

Comment:
Do we actually need this provider? It doesn't seem like it provides anything useful. The executable is already going to be in `DefaultInfo`.

Reply:
Good point. Removed.